task-2/routes/sum.js

```diff
@@ -0,0 +1,22 @@
+const { sendSuccessResponse, sendBadRequestResponse } = require('./../response-helper');
+
+function handler(request, response, url, data) {
+    const { a } = url.query;
+    const { b } = data;
+    const nA = Number(a);
+    const nB = Number(b);
+    if (isNaN(nA) || isNaN(nB)) {
```

Comment:
you can use `Number.isNaN`

Reply:
ahh, totally forgot about that, thank you

Reply:
ahh, totally forgot about that, thank you